src/main/java/com/ioet/bpm/skills/boundaries/SkillController.java

```diff
@@ -53,16 +53,12 @@ public SkillController(SkillRepository skillRepository) {
     @PutMapping("/{id}")
     public ResponseEntity<Skill> updateSkill(@PathVariable(value = "id") String skillId,
                                              @Valid @RequestBody Skill skillDetails) {
-
-        Optional<Skill> skillOptional = skillRepository.findById(skillId);
-        if (!skillOptional.isPresent()) {
+        Optional<Skill> skillFromDB = skillRepository.findById(skillId);
+        if (!skillFromDB.isPresent()) {
```

Comment:
My personal preference is it to check for isPresent and do the things we want and not to check if it is not present

Reply:
Let's follow your personal preferences. It was changed.